librz/core/cmd_analysis.c

```diff
@@ -8088,6 +8088,84 @@ RZ_IPI RzCmdStatus rz_analysis_rtti_demangle_class_name_handler(RzCore *core, in
 	return RZ_CMD_STATUS_ERROR;
 }
 
+RZ_IPI RzCmdStatus rz_analysis_print_global_variable_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
+	char *var_name = NULL;
```

Comment:
Can be solved by defining `var_name` as `const char*` instead.

Reply:
Ah, I forgot this..